packages/mini-apps-ui-kit-react/src/components/CountryDrawer/CountryDrawer.tsx

```diff
@@ -51,7 +54,9 @@ export function CountryDrawer({
     }
   }, [open]);
```

Comment:
@danielppolo try `autofocus` prop on search bar instead. BTW this didn't worked for me